### DISCUSSION TITLE
Remove students from PrairieLearn course?

### DISCUSSION THREAD

**Discussion by firasm** [PrairieLearn/PrairieLearn#6038] [Ideas]
Is there a way to remove students that accidentally registered for the wrong PrairieLearn course?

I accidentally pointed them to the wrong link in Canvas and now they're registered in a course I don't want them to be in.

Update: Tracking feature request in #6039.

**Comment by echuber2** [comment]
As long as students are able to add themselves by self-service (the default), I think your only option may be to use an explicit list of user IDs on the course instance access rules.

**Reply by firasm** [reply]
haha that sounds super fun. Okay maybe I need to create a new course instance and "retire" the one where there's a mix of students.

Thanks!

**Comment by nwalters512** [comment]
Is the "wrong" course your own course, or someone else's course? I don't think we have a great answer either way, but I think this could turn into a feature request for instructors being able to remove students from their own course (without resorting to a hardcoded list of allowed students).

**Reply by firasm** [reply]
It's my own course so I have full permissions.

**Reply by firasm** [reply]
Thanks, I've created a feature request in #6039.
